tasks/main.yml

```diff
@@ -1,10 +1,4 @@
 ---
-- name: Install package
-  become: True
-  package:
-    name: kubernetes-hyperkube-kube-apiserver
```

Comment:
While not strictly necessary - could we install the `*server` package here? Just to be sure that its always installed, even when this were to be the first role on a node.